packages/vuetify/src/components/VFileInput/VFileInput.tsx

```diff
@@ -105,15 +106,13 @@ export const VFileInput = defineComponent({
 
     useRender(() => {
       const hasCounter = !!(slots.counter || props.counter || counterValue.value)
-      const [_, restAttrs] = pick(attrs, ['class'])
+      const [rootAttrs, inputAttrs] = filterInputAttrs(attrs)
+      const [fieldProps, _] = filterFieldProps(props)
```

Comment:
This kinda sucks performance-wise, it would be simpler to just remove `makeVFieldProps()` from these components and use fallthrough instead. The downsides are that these props would no longer be automatically added to the type definitions or docs.